systems/plants/collision/DLLEXPORT_drakeCollision.h

```diff
@@ -0,0 +1,10 @@
+#undef DLLEXPORT_drakeCollision
```

Comment:
Why? This file exists to contain the DLLEXPORT_drakeCollision macro so that I don't have it copied multiple places.